### ISSUE TITLE
CSL is not defined

### ISSUE THREAD

**Issue by LemuelCushing** [larsgw/citation.js#27]
Hi, hope this is an appropriate question, being an utter newb. Good chance I'm just missing something obvious, but I can't figure out what..

Trying to get it run from the browser, I get the following error:
`Uncaught ReferenceError: CSL is not defined
    at citation-0.2.js:2487`
Which corresponds to:
```
  , window: {}
  , wdk: require( 'wikidata-sdk' )
  } : {
    CSL: CSL
  , striptags: function ( html ) {
      var tmp = document.createElement( 'div' )
      tmp.innerHTML = html
      return tmp.textContent || tmp.innerText || ''
    }
```
The CLI interface works fine, though I had to first pass the package through the `wrap-cmd` package to get it create a .cmd file on my Windows machine. (Which could very well be another mistaken understanding on my part.. =S )
I'm trying to incorporate it into my Jekyll page, which already parses a CSL-JSON file into a rudimentary bibliography.

Thank you for your work!
Lem

**Comment by larsgw** [comment]
It definitely is appropriate! The current setup for using Citation.js in the browser is a bit awkward at times, and this is a good reminder. The solution is in the README, but [not in the most obvious place](https://github.com/larsgw/citation.js#dependencies-1) (something that I should fix as well).

 **TL;DR:** You need to load [citeproc-js]() as well:

    <script src="path/to/citeproc.js" type="text/javascript"></script>
    <script src="path/to/citation-0.2.js" type="text/javascript"></script>

As I said, this is a bit clumsy, but should be fixed with #24, which I'll fix somewhere today or next week, I hope.

As for the CLI interface: Installing Citation.js [globally](https://docs.npmjs.com/getting-started/installing-npm-packages-globally) with `-g` or `--global` should do the trick.

**Comment by LemuelCushing** [comment]
Incredible, thanks! Don't know how I've missed it, felt like I scoured the docs.  

And regarding the CLI - though I have a suspicion that I'm missing something small but essential in my understanding of the nodejs environment, many (but not all) of the packages I've installed globally failed to load without the .cmd file, which the npm i -g doesn't always create. 
But that's neither here nor there. The point is, pretty sure I installed it globally, but I had to create a .cmd file for it to be initiated from the shell. 

Thanks again, for your help and your prompt reply!
Lem

**Comment by larsgw** [comment]
I know what the issue with the CLI is now. I need to actually put a `bin` entry in the `package.json`. I thought it did that automatically, but I guess not. Anyway, it should be fixed in the next version, or the first patches after the next version.
